src/test/java/de/zalando/aruha/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -164,7 +165,7 @@ public void whenTopicCreationFailsRemoveEventTypeFromRepositoryAnd500() throws E
     @Test
     public void whenPUTInvalidEventTypeThen422() throws Exception {
         EventType invalidEventType = buildEventType();
-        invalidEventType.setCategory("");
+        invalidEventType.setCategory(EventCategory.UNDEFINED);
```

Comment:
Is this correct? The test expects a problem with error "may not be empty" for the category field, which you just defined above.